src/coreclr/jit/importer.cpp

```diff
@@ -21299,6 +21300,7 @@ void Compiler::impDevirtualizeCall(GenTreeCall*            call,
     // it's a union field used for other things by virtual
     // stubs)
     call->gtInlineCandidateInfo = nullptr;
+    call->gtCallMoreFlags &= ~GTF_CALL_M_LATE_DEVIRT;
```

Comment:
Maybe we should rename this flag? `GTF_CALL_M_HAS_LATE_DEVIRT_INFO` perhaps? I introduced it but the name could better describe what it means.

Reply:
Done.